pyramid_zipkin/request_helper.py

```diff
@@ -119,6 +119,9 @@ def create_zipkin_attr(request):
     Attaches lazy attribute `zipkin_trace_id` with request which is then used
     throughout the tween.
 
+    consumes custom is_tracing function to determine if the request is traced
```

Comment:
Sorry to be nitpicky, but I prefer to keep docstrings nicely structured where possible. Could you capitalize and punctuate this like the rest of this docstring.